en/chromebook_setup/instructions.md

```diff
@@ -37,29 +44,40 @@ stuff useful computer code into for a project we're working on. We use them to
 keep the various bits of code we want for our various projects separate so
 things don't get mixed up between projects.
 
-In your terminal at the bottom of the Cloud 9 interface, run the following:
+In your terminal, type the following lines. Hit enter after each one to send the
+instruction to the computer.
 
-{% filename %}Cloud 9{% endfilename %}
+{% filename %}CodeAnywhere{% endfilename %}
 ```
 sudo apt update
-sudo apt install python3.6-venv
+sudo apt install python3.4-venv
```

Comment:
I was wondering why not use python3.6, but it doesn't seem to be available in that environment. Are there other places in the tutorial that refer to the specific Python version? Could using 3.4 instead cause issues in other steps?